### ISSUE TITLE
Crash when installing it on another computer

### ISSUE THREAD

**Issue by mnin** [rxhanson/RectanglePro-Community#148]
Maybe it got the notification (config is used on another machine) and crashed.

```
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               Rectangle Pro [681]
Path:                  /Applications/Rectangle Pro.app/Contents/MacOS/Rectangle Pro
Identifier:            com.knollsoft.Hookshot
Version:               2.6.3 (125)
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2022-10-26 19:09:04.5878 +0200
OS Version:            macOS 12.6 (21G115)
Report Version:        12
Anonymous UUID:        974A243D-BEF3-0DAC-532F-18B7E4F28DF2


Time Awake Since Boot: 34000 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BREAKPOINT (SIGTRAP)
Exception Codes:       0x0000000000000001, 0x00000001008dd1f4
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    Namespace SIGNAL, Code 5 Trace/BPT trap: 5
Terminating Process:   exc handler [681]

Thread 0 Crashed::  Dispatch queue: com.apple.main-thread
0   Rectangle Pro                 	       0x1008dd1f4 0x100848000 + 610804
1   Rectangle Pro                 	       0x1008dcf74 0x100848000 + 610164
2   Rectangle Pro                 	       0x100916348 0x100848000 + 844616
3   Rectangle Pro                 	       0x1008faa40 0x100848000 + 731712
4   Foundation                    	       0x1bbacb758 -[__NSObserver _doit:] + 348
5   CoreFoundation                	       0x1bac09468 __CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER__ + 28
6   CoreFoundation                	       0x1bacadbb8 ___CFXRegistrationPost_block_invoke + 52
7   CoreFoundation                	       0x1bacadb24 _CFXRegistrationPost + 456
8   CoreFoundation                	       0x1babd80f8 _CFXNotificationPost + 732
9   Foundation                    	       0x1bba9dc28 -[NSNotificationCenter postNotificationName:object:userInfo:] + 96
10  Rectangle Pro                 	       0x1008fa8b0 0x100848000 + 731312
11  Rectangle Pro                 	       0x1008cd014 0x100848000 + 544788
12  Rectangle Pro                 	       0x1008c347c 0x100848000 + 504956
13  Rectangle Pro                 	       0x1008cd404 0x100848000 + 545796
14  Rectangle Pro                 	       0x1008c1c94 0x100848000 + 498836
15  libdispatch.dylib             	       0x1ba992fb4 _dispatch_block_async_invoke2 + 148
16  libdispatch.dylib             	       0x1ba9841b4 _dispatch_client_callout + 20
17  libdispatch.dylib             	       0x1ba987670 _dispatch_continuation_pop + 500
18  libdispatch.dylib             	       0x1ba99a8e0 _dispatch_source_invoke + 1596
19  libdispatch.dylib             	       0x1ba992620 _dispatch_main_queue_drain + 756
20  libdispatch.dylib             	       0x1ba99231c _dispatch_main_queue_callback_4CF + 44
21  CoreFoundation                	       0x1bac56998 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 16
22  CoreFoundation                	       0x1bac13c08 __CFRunLoopRun + 2532
23  CoreFoundation                	       0x1bac12a84 CFRunLoopRunSpecific + 600
24  HIToolbox                     	       0x1c3852338 RunCurrentEventLoopInMode + 292
25  HIToolbox                     	       0x1c38520b4 ReceiveNextEventCommon + 564
26  HIToolbox                     	       0x1c3851e68 _BlockUntilNextEventMatchingListInModeWithFilter + 72
27  AppKit                        	       0x1bd77a51c _DPSNextEvent + 860
28  AppKit                        	       0x1bd778e14 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1328
29  AppKit                        	       0x1bd76afe0 -[NSApplication run] + 596
30  AppKit                        	       0x1bd73c6fc NSApplicationMain + 1132
31  Rectangle Pro                 	       0x1008529cc 0x100848000 + 43468
32  dyld                          	       0x100a4908c start + 520

Thread 1:: com.apple.NSEventThread
0   libsystem_kernel.dylib        	       0x1bab0a8b0 mach_msg_trap + 8
1   libsystem_kernel.dylib        	       0x1bab0ad20 mach_msg + 76
2   CoreFoundation                	       0x1bac15210 __CFRunLoopServiceMachPort + 372
3   CoreFoundation                	       0x1bac136c0 __CFRunLoopRun + 1180
4   CoreFoundation                	       0x1bac12a84 CFRunLoopRunSpecific + 600
5   AppKit                        	       0x1bd8e7100 _NSEventThread + 196
6   libsystem_pthread.dylib       	       0x1bab4826c _pthread_start + 148
7   libsystem_pthread.dylib       	       0x1bab4308c thread_start + 8

Thread 2:: com.apple.NSURLConnectionLoader
0   libsystem_kernel.dylib        	       0x1bab0a8b0 mach_msg_trap + 8
1   libsystem_kernel.dylib        	       0x1bab0ad20 mach_msg + 76
2   CoreFoundation                	       0x1bac15210 __CFRunLoopServiceMachPort + 372
3   CoreFoundation                	       0x1bac136c0 __CFRunLoopRun + 1180
4   CoreFoundation                	       0x1bac12a84 CFRunLoopRunSpecific + 600
5   CFNetwork                     	       0x1bfa0a638 0x1bf79f000 + 2537016
6   Foundation                    	       0x1bbaf05cc __NSThread__start__ + 808
7   libsystem_pthread.dylib       	       0x1bab4826c _pthread_start + 148
8   libsystem_pthread.dylib       	       0x1bab4308c thread_start + 8

Thread 3:
0   libsystem_pthread.dylib       	       0x1bab43078 start_wqthread + 0

Thread 4:
0   libsystem_pthread.dylib       	       0x1bab43078 start_wqthread + 0

Thread 5:
0   libsystem_pthread.dylib       	       0x1bab43078 start_wqthread + 0


Thread 0 crashed with ARM Thread State (64-bit):
    x0: 0x000000016f5b4f30   x1: 0x000000012b727630   x2: 0x0000000000000000   x3: 0x000000016f5b4f58
    x4: 0x0000000000000000   x5: 0x000000016f5b4d4c   x6: 0x0000000000000000   x7: 0x0000000000000000
    x8: 0x000000007fffffff   x9: 0x0000000000000028  x10: 0x0000000c00000003  x11: 0x0000000e00000003
   x12: 0x0000000000000002  x13: 0x000000011a81f430  x14: 0x0000000084e97800  x15: 0x00000002135a5450
   x16: 0x00000001bab5f0d0  x17: 0x000000021433f910  x18: 0x0000000000000000  x19: 0x000000012b7275e0
   x20: 0x000000012b727608  x21: 0x0000000000000000  x22: 0x0000000204eba3b1  x23: 0x0000000214357368
   x24: 0x0000000100921358  x25: 0x0000600003ab5ce0  x26: 0x000000b9000000ad  x27: 0x0000000000001c00
   x28: 0x0000000000000000   fp: 0x000000016f5b4fb0   lr: 0x00000001008dcf74
    sp: 0x000000016f5b4ef0   pc: 0x00000001008dd1f4 cpsr: 0x60001000
   far: 0x0000600003a70000  esr: 0xf2000001 (Breakpoint) brk 1

Binary Images:
       0x100848000 -        0x100987fff com.knollsoft.Hookshot (2.6.3) <55e8caee-5535-3283-84ea-b5a0b47d8642> /Applications/Rectangle Pro.app/Contents/MacOS/Rectangle Pro
       0x1bba94000 -        0x1bbe85fff com.apple.Foundation (6.9) <8bbf8f9b-0034-34f0-a2c5-13c31941632d> /System/Library/Frameworks/Foundation.framework/Versions/C/Foundation
       0x1bab90000 -        0x1bb0d6fff com.apple.CoreFoundation (6.9) <fc3c193d-0cdb-3569-9f0e-bd2507ca1dbb> /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
       0x1ba980000 -        0x1ba9c6fff libdispatch.dylib (*) <b3c7a004-1069-3171-b630-2c386a8b399c> /usr/lib/system/libdispatch.dylib
       0x1c3820000 -        0x1c3b53fff com.apple.HIToolbox (2.1.1) <aaf900bd-bfb6-3af0-a8d3-e24bbe1d57f5> /System/Library/Frameworks/Carbon.framework/Versions/A/Frameworks/HIToolbox.framework/Versions/A/HIToolbox
       0x1bd739000 -        0x1be5f1fff com.apple.AppKit (6.9) <5e432f87-5b58-391a-a542-fa2d909dd210> /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit
       0x100a44000 -        0x100aa3fff dyld (*) <38ee9fe9-b66d-3066-8c5c-6ddf0d6944c6> /usr/lib/dyld
       0x1bab09000 -        0x1bab40fff libsystem_kernel.dylib (*) <a9d87740-9c1d-3468-bf60-720a8d713cba> /usr/lib/system/libsystem_kernel.dylib
       0x1bab41000 -        0x1bab4dfff libsystem_pthread.dylib (*) <63c4eef9-69a5-38b1-996e-8d31b66a051d> /usr/lib/system/libsystem_pthread.dylib
       0x1bf79f000 -        0x1bfc53fff com.apple.CFNetwork (1335.0.3) <ba0ade10-aa7e-3168-a6c6-817b01849dd8> /System/Library/Frameworks/CFNetwork.framework/Versions/A/CFNetwork

External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0

VM Region Summary:
ReadOnly portion of Libraries: Total=1.0G resident=0K(0%) swapped_out_or_unallocated=1.0G(100%)
Writable regions: Total=1.9G written=0K(0%) resident=0K(0%) swapped_out=0K(0%) unallocated=1.9G(100%)

                                VIRTUAL   REGION 
REGION TYPE                        SIZE    COUNT (non-coalesced) 
===========                     =======  ======= 
Accelerate framework               640K        5 
Activity Tracing                   256K        1 
ColorSync                          576K       25 
CoreAnimation                      624K       17 
CoreGraphics                        48K        3 
CoreUI image data                 1216K        8 
Foundation                          16K        1 
IOKit                              128K        8 
Kernel Alloc Once                   32K        1 
MALLOC                           318.5M       64 
MALLOC guard page                  288K       15 
MALLOC_MEDIUM (reserved)           1.2G       10         reserved VM address space (unallocated)
MALLOC_NANO (reserved)           384.0M        1         reserved VM address space (unallocated)
SQLite page cache                  192K        3 
STACK GUARD                       56.1M        6 
Stack                             10.6M        6 
VM_ALLOCATE                        112K        7 
__AUTH                            2263K      239 
__AUTH_CONST                      17.7M      421 
__CTF                               756        1 
__DATA                            11.3M      413 
__DATA_CONST                      15.5M      429 
__DATA_DIRTY                      1460K      170 
__FONT_DATA                          4K        1 
__LINKEDIT                       578.6M        7 
__OBJC_CONST                      2650K      210 
__OBJC_RO                         83.0M        1 
__OBJC_RW                         3168K        1 
__TEXT                           431.5M      446 
__UNICODE                          592K        1 
dyld private memory               1024K        1 
libnetwork                         128K        8 
mapped file                      143.2M       26 
shared memory                      864K       13 
===========                     =======  ======= 
TOTAL                              3.2G     2569 
TOTAL, minus reserved VM space     1.6G     2569 



-----------
Full Report
-----------

{"app_name":"Rectangle Pro","timestamp":"2022-10-26 19:09:06.00 +0200","app_version":"2.6.3","slice_uuid":"55e8caee-5535-3283-84ea-b5a0b47d8642","build_version":"125","platform":1,"bundleID":"com.knollsoft.Hookshot","share_with_app_devs":0,"is_first_party":0,"bug_type":"309","os_version":"macOS 12.6 (21G115)","incident_id":"416EE0EB-5274-4E81-AC7A-1385129BD4D0","name":"Rectangle Pro"}
{
  "uptime" : 34000,
  "procLaunch" : "2022-10-26 09:34:56.1538 +0200",
  "procRole" : "Default",
  "version" : 2,
  "userID" : 501,
  "deployVersion" : 210,
  "modelCode" : "MacBookPro18,2",
  "procStartAbsTime" : 902789309,
  "coalitionID" : 775,
  "osVersion" : {
    "train" : "macOS 12.6",
    "build" : "21G115",
    "releaseType" : "User"
  },
  "captureTime" : "2022-10-26 19:09:04.5878 +0200",
  "incident" : "416EE0EB-5274-4E81-AC7A-1385129BD4D0",
  "bug_type" : "309",
  "pid" : 681,
  "procExitAbsTime" : 827663093018,
  "translated" : false,
  "cpuType" : "ARM-64",
  "procName" : "Rectangle Pro",
  "procPath" : "\/Applications\/Rectangle Pro.app\/Contents\/MacOS\/Rectangle Pro",
  "bundleInfo" : {"CFBundleShortVersionString":"2.6.3","CFBundleVersion":"125","CFBundleIdentifier":"com.knollsoft.Hookshot"},
  "storeInfo" : {"deviceIdentifierForVendor":"4F8C5E1F-E56F-5E14-9A6D-DC80350F0EBC","thirdParty":true},
  "parentProc" : "launchd",
  "parentPid" : 1,
  "coalitionName" : "com.knollsoft.Hookshot",
  "crashReporterKey" : "974A243D-BEF3-0DAC-532F-18B7E4F28DF2",
  "sip" : "enabled",
  "isCorpse" : 1,
  "exception" : {"codes":"0x0000000000000001, 0x00000001008dd1f4","rawCodes":[1,4304261620],"type":"EXC_BREAKPOINT","signal":"SIGTRAP"},
  "termination" : {"flags":0,"code":5,"namespace":"SIGNAL","indicator":"Trace\/BPT trap: 5","byProc":"exc handler","byPid":681},
  "extMods" : {"caller":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"system":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"targeted":{"thread_create":0,"thread_set_state":0,"task_for_pid":0},"warnings":0},
  "faultingThread" : 0,
  "threads" : [{"triggered":true,"id":6276,"threadState":{"x":[{"value":6163222320},{"value":5023888944},{"value":0},{"value":6163222360},{"value":0},{"value":6163221836},{"value":0},{"value":0},{"value":2147483647},{"value":40},{"value":51539607555},{"value":60129542147},{"value":2},{"value":4739691568},{"value":2229893120},{"value":8914621520,"symbolLocation":0,"symbol":"OBJC_CLASS_$___NSCFString"},{"value":7427453136,"symbolLocation":0,"symbol":"_platform_memmove"},{"value":8928885008},{"value":0},{"value":5023888864},{"value":5023888904},{"value":0},{"value":8672486321,"objc-selector":"defaultCenter"},{"value":8928981864,"symbolLocation":0,"symbol":"value witness table for Notification"},{"value":4304540504},{"value":105553177828576},{"value":794568949933},{"value":7168},{"value":0}],"flavor":"ARM_THREAD_STATE64","lr":{"value":4304260980},"cpsr":{"value":1610616832},"fp":{"value":6163222448},"sp":{"value":6163222256},"esr":{"value":4060086273,"description":"(Breakpoint) brk 1"},"pc":{"value":4304261620,"matchesCrashFrame":1},"far":{"value":105553177542656}},"queue":"com.apple.main-thread","frames":[{"imageOffset":610804,"imageIndex":0},{"imageOffset":610164,"imageIndex":0},{"imageOffset":844616,"imageIndex":0},{"imageOffset":731712,"imageIndex":0},{"imageOffset":227160,"symbol":"-[__NSObserver _doit:]","symbolLocation":348,"imageIndex":1},{"imageOffset":496744,"symbol":"__CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER__","symbolLocation":28,"imageIndex":2},{"imageOffset":1170360,"symbol":"___CFXRegistrationPost_block_invoke","symbolLocation":52,"imageIndex":2},{"imageOffset":1170212,"symbol":"_CFXRegistrationPost","symbolLocation":456,"imageIndex":2},{"imageOffset":295160,"symbol":"_CFXNotificationPost","symbolLocation":732,"imageIndex":2},{"imageOffset":39976,"symbol":"-[NSNotificationCenter postNotificationName:object:userInfo:]","symbolLocation":96,"imageIndex":1},{"imageOffset":731312,"imageIndex":0},{"imageOffset":544788,"imageIndex":0},{"imageOffset":504956,"imageIndex":0},{"imageOffset":545796,"imageIndex":0},{"imageOffset":498836,"imageIndex":0},{"imageOffset":77748,"symbol":"_dispatch_block_async_invoke2","symbolLocation":148,"imageIndex":3},{"imageOffset":16820,"symbol":"_dispatch_client_callout","symbolLocation":20,"imageIndex":3},{"imageOffset":30320,"symbol":"_dispatch_continuation_pop","symbolLocation":500,"imageIndex":3},{"imageOffset":108768,"symbol":"_dispatch_source_invoke","symbolLocation":1596,"imageIndex":3},{"imageOffset":75296,"symbol":"_dispatch_main_queue_drain","symbolLocation":756,"imageIndex":3},{"imageOffset":74524,"symbol":"_dispatch_main_queue_callback_4CF","symbolLocation":44,"imageIndex":3},{"imageOffset":813464,"symbol":"__CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__","symbolLocation":16,"imageIndex":2},{"imageOffset":539656,"symbol":"__CFRunLoopRun","symbolLocation":2532,"imageIndex":2},{"imageOffset":535172,"symbol":"CFRunLoopRunSpecific","symbolLocation":600,"imageIndex":2},{"imageOffset":205624,"symbol":"RunCurrentEventLoopInMode","symbolLocation":292,"imageIndex":4},{"imageOffset":204980,"symbol":"ReceiveNextEventCommon","symbolLocation":564,"imageIndex":4},{"imageOffset":204392,"symbol":"_BlockUntilNextEventMatchingListInModeWithFilter","symbolLocation":72,"imageIndex":4},{"imageOffset":267548,"symbol":"_DPSNextEvent","symbolLocation":860,"imageIndex":5},{"imageOffset":261652,"symbol":"-[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:]","symbolLocation":1328,"imageIndex":5},{"imageOffset":204768,"symbol":"-[NSApplication run]","symbolLocation":596,"imageIndex":5},{"imageOffset":14076,"symbol":"NSApplicationMain","symbolLocation":1132,"imageIndex":5},{"imageOffset":43468,"imageIndex":0},{"imageOffset":20620,"symbol":"start","symbolLocation":520,"imageIndex":6}]},{"id":6417,"name":"com.apple.NSEventThread","frames":[{"imageOffset":6320,"symbol":"mach_msg_trap","symbolLocation":8,"imageIndex":7},{"imageOffset":7456,"symbol":"mach_msg","symbolLocation":76,"imageIndex":7},{"imageOffset":545296,"symbol":"__CFRunLoopServiceMachPort","symbolLocation":372,"imageIndex":2},{"imageOffset":538304,"symbol":"__CFRunLoopRun","symbolLocation":1180,"imageIndex":2},{"imageOffset":535172,"symbol":"CFRunLoopRunSpecific","symbolLocation":600,"imageIndex":2},{"imageOffset":1761536,"symbol":"_NSEventThread","symbolLocation":196,"imageIndex":5},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":8},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":8}]},{"id":6532,"name":"com.apple.NSURLConnectionLoader","frames":[{"imageOffset":6320,"symbol":"mach_msg_trap","symbolLocation":8,"imageIndex":7},{"imageOffset":7456,"symbol":"mach_msg","symbolLocation":76,"imageIndex":7},{"imageOffset":545296,"symbol":"__CFRunLoopServiceMachPort","symbolLocation":372,"imageIndex":2},{"imageOffset":538304,"symbol":"__CFRunLoopRun","symbolLocation":1180,"imageIndex":2},{"imageOffset":535172,"symbol":"CFRunLoopRunSpecific","symbolLocation":600,"imageIndex":2},{"imageOffset":2537016,"imageIndex":9},{"imageOffset":378316,"symbol":"__NSThread__start__","symbolLocation":808,"imageIndex":1},{"imageOffset":29292,"symbol":"_pthread_start","symbolLocation":148,"imageIndex":8},{"imageOffset":8332,"symbol":"thread_start","symbolLocation":8,"imageIndex":8}]},{"id":376255,"frames":[{"imageOffset":8312,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":8}]},{"id":380346,"frames":[{"imageOffset":8312,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":8}]},{"id":380347,"frames":[{"imageOffset":8312,"symbol":"start_wqthread","symbolLocation":0,"imageIndex":8}]}],
  "usedImages" : [
  {
    "source" : "P",
    "arch" : "arm64",
    "base" : 4303650816,
    "CFBundleShortVersionString" : "2.6.3",
    "CFBundleIdentifier" : "com.knollsoft.Hookshot",
    "size" : 1310720,
    "uuid" : "55e8caee-5535-3283-84ea-b5a0b47d8642",
    "path" : "\/Applications\/Rectangle Pro.app\/Contents\/MacOS\/Rectangle Pro",
    "name" : "Rectangle Pro",
    "CFBundleVersion" : "125"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 7443398656,
    "CFBundleShortVersionString" : "6.9",
    "CFBundleIdentifier" : "com.apple.Foundation",
    "size" : 4136960,
    "uuid" : "8bbf8f9b-0034-34f0-a2c5-13c31941632d",
    "path" : "\/System\/Library\/Frameworks\/Foundation.framework\/Versions\/C\/Foundation",
    "name" : "Foundation",
    "CFBundleVersion" : "1866"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 7427653632,
    "CFBundleShortVersionString" : "6.9",
    "CFBundleIdentifier" : "com.apple.CoreFoundation",
    "size" : 5533696,
    "uuid" : "fc3c193d-0cdb-3569-9f0e-bd2507ca1dbb",
    "path" : "\/System\/Library\/Frameworks\/CoreFoundation.framework\/Versions\/A\/CoreFoundation",
    "name" : "CoreFoundation",
    "CFBundleVersion" : "1866"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 7425490944,
    "size" : 290816,
    "uuid" : "b3c7a004-1069-3171-b630-2c386a8b399c",
    "path" : "\/usr\/lib\/system\/libdispatch.dylib",
    "name" : "libdispatch.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 7575044096,
    "CFBundleShortVersionString" : "2.1.1",
    "CFBundleIdentifier" : "com.apple.HIToolbox",
    "size" : 3358720,
    "uuid" : "aaf900bd-bfb6-3af0-a8d3-e24bbe1d57f5",
    "path" : "\/System\/Library\/Frameworks\/Carbon.framework\/Versions\/A\/Frameworks\/HIToolbox.framework\/Versions\/A\/HIToolbox",
    "name" : "HIToolbox"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 7473434624,
    "CFBundleShortVersionString" : "6.9",
    "CFBundleIdentifier" : "com.apple.AppKit",
    "size" : 15437824,
    "uuid" : "5e432f87-5b58-391a-a542-fa2d909dd210",
    "path" : "\/System\/Library\/Frameworks\/AppKit.framework\/Versions\/C\/AppKit",
    "name" : "AppKit",
    "CFBundleVersion" : "2113.60.148"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 4305731584,
    "size" : 393216,
    "uuid" : "38ee9fe9-b66d-3066-8c5c-6ddf0d6944c6",
    "path" : "\/usr\/lib\/dyld",
    "name" : "dyld"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 7427100672,
    "size" : 229376,
    "uuid" : "a9d87740-9c1d-3468-bf60-720a8d713cba",
    "path" : "\/usr\/lib\/system\/libsystem_kernel.dylib",
    "name" : "libsystem_kernel.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 7427330048,
    "size" : 53248,
    "uuid" : "63c4eef9-69a5-38b1-996e-8d31b66a051d",
    "path" : "\/usr\/lib\/system\/libsystem_pthread.dylib",
    "name" : "libsystem_pthread.dylib"
  },
  {
    "source" : "P",
    "arch" : "arm64e",
    "base" : 7507406848,
    "CFBundleShortVersionString" : "1335.0.3",
    "CFBundleIdentifier" : "com.apple.CFNetwork",
    "size" : 4935680,
    "uuid" : "ba0ade10-aa7e-3168-a6c6-817b01849dd8",
    "path" : "\/System\/Library\/Frameworks\/CFNetwork.framework\/Versions\/A\/CFNetwork",
    "name" : "CFNetwork",
    "CFBundleVersion" : "1335.0.3"
  }
],
  "sharedCache" : {
  "base" : 7424032768,
  "size" : 3144744960,
  "uuid" : "049b583e-d4dc-3b2b-88da-e52d6e18313d"
},
  "vmSummary" : "ReadOnly portion of Libraries: Total=1.0G resident=0K(0%) swapped_out_or_unallocated=1.0G(100%)\nWritable regions: Total=1.9G written=0K(0%) resident=0K(0%) swapped_out=0K(0%) unallocated=1.9G(100%)\n\n                                VIRTUAL   REGION \nREGION TYPE                        SIZE    COUNT (non-coalesced) \n===========                     =======  ======= \nAccelerate framework               640K        5 \nActivity Tracing                   256K        1 \nColorSync                          576K       25 \nCoreAnimation                      624K       17 \nCoreGraphics                        48K        3 \nCoreUI image data                 1216K        8 \nFoundation                          16K        1 \nIOKit                              128K        8 \nKernel Alloc Once                   32K        1 \nMALLOC                           318.5M       64 \nMALLOC guard page                  288K       15 \nMALLOC_MEDIUM (reserved)           1.2G       10         reserved VM address space (unallocated)\nMALLOC_NANO (reserved)           384.0M        1         reserved VM address space (unallocated)\nSQLite page cache                  192K        3 \nSTACK GUARD                       56.1M        6 \nStack                             10.6M        6 \nVM_ALLOCATE                        112K        7 \n__AUTH                            2263K      239 \n__AUTH_CONST                      17.7M      421 \n__CTF                               756        1 \n__DATA                            11.3M      413 \n__DATA_CONST                      15.5M      429 \n__DATA_DIRTY                      1460K      170 \n__FONT_DATA                          4K        1 \n__LINKEDIT                       578.6M        7 \n__OBJC_CONST                      2650K      210 \n__OBJC_RO                         83.0M        1 \n__OBJC_RW                         3168K        1 \n__TEXT                           431.5M      446 \n__UNICODE                          592K        1 \ndyld private memory               1024K        1 \nlibnetwork                         128K        8 \nmapped file                      143.2M       26 \nshared memory                      864K       13 \n===========                     =======  ======= \nTOTAL                              3.2G     2569 \nTOTAL, minus reserved VM space     1.6G     2569 \n",
  "legacyInfo" : {
  "threadTriggered" : {
    "queue" : "com.apple.main-thread"
  }
},
  "trialInfo" : {
  "rollouts" : [
    {
      "rolloutId" : "61301e3a61217b3110231469",
      "factorPackIds" : {
        "SIRI_FIND_MY_CONFIGURATION_FILES" : "6348493aa52bb16adc4e4d06"
      },
      "deploymentId" : 240000023
    },
    {
      "rolloutId" : "60186475825c62000ccf5450",
      "factorPackIds" : {

      },
      "deploymentId" : 240000026
    }
  ],
  "experiments" : [

  ]
}
}
```

**Comment by rxhanson** [comment]
Thanks for the crash report here. I'll analyze it and put in a fix in an upcoming release.

**Comment by rxhanson** [comment]
Alright, the fix is in v2.7.1. Let me know if for some reason you hit another crash.
